deploy/.env.production.local.tpl

```diff
@@ -31,10 +31,12 @@
     "MinimumPeers": 4
   },
   "Datastore": {
-    "Dir": "/home/nonroot/datastore",
-    "Type": "levelds",
-    "TmpDir": "/home/nonroot/tmpstore",
-    "TmpType": "levelds"
+    "Type": "dynamodb",
+    "Dir": "${TF_WORKSPACE}-${TF_VAR_app}-datastore",
+    "Region": "$TF_VAR_region",
+    "TmpType": "dynamodb",
```

Comment:
Do we want the temporary datastore in dynamo? Why not just store it in `/tmp`?

Reply:
that's a good question. Honestly, I don't know what the temp datastore is used for. I'll take a look.

Reply:
I'll keep it in dynamo for now. It's probably ok to live in the container's ephemeral storage but I'm not sure what the space requirements are. We can revisit in the future and move if needed.